.eslintrc.json

```diff
@@ -4,6 +4,9 @@
   "overrides": [
     {
       "files": ["README.md"],
+      "parserOptions": {
+        "sourceType": "module"
+      },
```

Comment:
It might be possible to do this automatically in `sanctuary-scripts` in case `module-type = esm`.